website/docusaurus.config.js

```diff
@@ -52,7 +52,8 @@ module.exports = {
           }
         ]
       }
-    ]
+    ],
+    require.resolve("@easyops-cn/docusaurus-search-local"),
```

Comment:
why require.resolve instead of just 
```suggestion
    "@easyops-cn/docusaurus-search-local",
```
?

Reply:
Right, I copied it from the project README, but we don't need it :+1: